src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -90,6 +120,24 @@ public function getAccessToken(): string
     }
 
     /**
+     * Specification:
+     * - The method return link to your own code source provider system.
```

Comment:
Returns the link to the source code provider

Reply:
++